test/libsolidity/semanticTests/stdlib/sha256.sol

```diff
@@ -0,0 +1,13 @@
+import "std/precompiles.sol";
```

Comment:
Shouldn't this be `std/crypto.sol`? Does it matter if they are implemented using precompiles?

Reply:
I'm still contesting this change.

The idea behind `precompiles` was that this would be a very thing wrapper around the precompiles only, but not more. For useful abstractions over the precompiles I think we'd need to do much nicer interfaces than what is currently provided.

If we are happy changing then and dropping the current versions, then the `cryptography` naming is fine, but we can still only do it in breaking releases. However if we keep them under `precompiles` we can still easily in a non-breaking way introduce `cryptography`.